File: problem-1/prob1.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# author : Arnaud Gourlay

def calculation():
    result = 0
    for i in range (1,1000):
        if ( i%3 == 0 ) or ( i%5 == 0 ):
            result = result + i
    print "The sum of all the multiples of 3 or 5 below 1000 is %s" %result


if __name__ == '__main__':
    calculation()

File: 001/Prob1.java
public class Prob1 {

    public static void main(String[] args) {
		int result = 0;
		for (int i = 1; i <1000; i++ ){
			if (i % 3 == 0 || i % 5 == 0){
				result = result + i;
			}
		}
		System.out.println("The sum of all the multiples of 3 or 5 below 1000 is "+result);
	}
}